packages/calling-stateful-client/src/CallClientState.ts

```diff
@@ -457,6 +457,22 @@ export interface VideoStreamRendererViewState {
   target: HTMLElement;
 }
 
+/* @conditional-compile-remove(media-access) */
+/**
+ * Media access state
+ * @alpha
+ */
+export interface MediaAccessState {
+  /**
+   * is audio permitted
```

Comment:
nit: Let's have these documents start capitalized
`Is audio permitted` would also be nice to know what you mean is audio permitted. Like where is audio permitted? In the call?